pkg/output/tables/clusters.yaml

```diff
@@ -0,0 +1,44 @@
+#
+# Copyright (c) 2021 Red Hat, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+columns:
+  id:
+    header: ID
+    width: 32
+  name:
+    header: NAME
+    width: 28
+  api.url:
+    header: API URL
+    width: 58
+  openshift_version:
+    header: OPENSHIFT VERSION
+    width: 18
+  product.id:
+    header: PRODUCT
+    width: 14
+  cloud_provider.id:
+    header: CLOUD PROVIDER
+    width: 14
+  region.id:
+    header: REGION
+    width: 14
+  state:
+    header: STATE
+    width: 13
+  external_id:
+    header: EXTERNAL_ID
+    width: 36
```

Comment:
```suggestion
    width: 36

```

Reply:
Thanks @T0MASD, I will add this in the next version of the patch.